src/pages/Trainee/TraineeList.jsx

```diff
@@ -30,13 +48,33 @@ export default class Trainee extends React.Component {
   }
 
   render() {
+    const { classes } = this.props;
     const { open } = this.state;
     return (
-      <>
-        <Button type="button" color="primary" variant="outlined" onClick={this.handleClickOpen}>
-          Add Trainee
-        </Button>
+      <div className={classes.paper}>
+        <div className={classes.buttonPosition}>
+          <Button type="button" color="primary" className={classes.button} variant="outlined" onClick={this.handleClickOpen}>
+            Add TraineeList
+          </Button>
+        </div>
         <FormDialog open={open} onClose={this.handleClickClose} onSubmit={() => this.handleSubmit} />
+        <Table
+          id="id"
```

Comment:
The value of "id" is only an example in the “table-props.png”.
Please use a unique id for table component like "table", because if you need to give "id" to multiple components than it must be unique.